src/inventory/player_inventory.rs

```diff
@@ -358,12 +364,23 @@ impl Inventory for PlayerInventory {
         }
     }
 
-    fn close(&mut self) {
```

Comment:
Are you sure that there is no need for inventories to handle them being closed specially?

Reply:
Okay, fair enough, i guess we can always readd the `close` method later on if we have to